### PR TITLE
Add vine

### PR DESCRIPTION
**Pull request by frol**
Vine is a module implementing promises for Python. It is a relatively new dependency for Py-AMQP -> Celery.

### CODE REVIEW

**Comment by frol** [comment]
@conda-forge/core It is ready for merge.

File: recipes/vine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - vine
+
+about:
+  home: http://github.com/celery/vine
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Python promises'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (jakirkham) [inline recipes/vine/meta.yaml]: Please drop this comment from the `example`.

**Comment by jakirkham** [comment]
One minor comment, then LGTM.

Note: Restarted Travis CI after the fixing issue ( https://github.com/conda-forge/staged-recipes/issues/2498 ).

File: recipes/vine/meta.yaml (hunk 1 above)
Comment (frol) [inline recipes/vine/meta.yaml]: Dropped.

**Comment by jakirkham** [comment]
Thanks @frol.

**Pull request merged**